docs/Tags.rst

```diff
@@ -42,7 +42,7 @@ for the tag assignment spreadsheet.
 - `labors <labors-tag-index>`: Tools that deal with labor assignment.
 - `map <map-tag-index>`: Tools that interact with the game map.
 - `military <military-tag-index>`: Tools that interact with the military.
-- `plants <plants-tag-index>`: Tools that interact with trees, shrubs, and crops.
+- `plants <plants-tag-index>`: Tools that interact with grass, trees, shrubs, and crops.
```

Comment:
I updated https://docs.google.com/spreadsheets/d/1hiDlo8M_bB_1jE-5HRs2RrrA_VZ4cRu9VXaTctX_nwk/edit#gid=170388995 to match this